moped-editor/src/views/staff/EditStaffView.js

```diff
@@ -93,6 +105,44 @@ const EditStaffView = () => {
                       validationSchema={validationSchema}
                       userCognitoId={userCognitoId}
                       isUserActive={isUserActive}
+                      isPasswordFieldDisabled={false}
+                      roleOptions={
+                        isNonLoginUser
+                          ? existingNonLoginUserRoleOptions
+                          : existingLoginUserRoleOptions
+                      }
+                      FormButtons={({
+                        isSubmitting,
+                        watch,
+                        handleCloseModal,
+                        setModalState,
+                      }) => (
+                        <>
+                          {!isNonLoginUser && (
+                            <StaffFormSaveButton disabled={isSubmitting} />
```

Comment:
@mddilley i'm guessing it's non-trivial to implement this, but it would be nice of the save button were disabled until some field value changed. 

rfh's `formState` object has an [`isdirty`](https://react-hook-form.com/api/useform/formstate) prop that you can use to determine that, but i'm not sure how easily you can get that value into this `disabled` prop, e.g. `disabled={isSubmitting || !isDirty}`. i'm happy to create a separate issue for it.

Reply:
This would be a great improvement, and I think that you are spot on about how to implement this. Could be a good snackoo issue. 🍫